api/validators/MerchStoreRequests.ts

```diff
@@ -129,6 +131,29 @@ export class MerchItemOptionEdit implements IMerchItemOptionEdit {
   metadata?: MerchItemOptionMetadata;
 }
 
+// TODO: use global variable for limit
+export class MerchItemPhoto implements IMerchItemPhoto {
+  @Allow()
+  picture: string;
+
+  @Min(0)
+  @Max(4)
+  position: number;
+}
```

Comment:
Given our descoping of positioning logic, I don't think we need any constraints on this position number. I think it makes sense to let position be any number possible, and have the frontend manage whatever number they want them to be, since the only thing that matters is their relative order to each other. e.g., 0, 1, 2, 3 conveys the same thing as -12091, -234, 394, 9023840932.